### PR TITLE
Creating new RHEL 7 STIG GUI profile

### PR DESCRIPTION
**Pull request by carlosmmatos**
#### Description:

New RHEL 7 STIG with GUI profile. Tackles issues that allow existing customers to utilize the RHEL 7 STIG during install, and still maintain a graphical env if they chose to have one.

### CODE REVIEW

File: rhel7/profiles/stig_gui.profile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+metadata:
+    version: V3R1
```
Comment (yuumasato) [inline rhel7/profiles/stig_gui.profile:4]: Update the version in the metadata too.

Comment (yuumasato) [reply rhel7/profiles/stig_gui.profile:4]: It is still V1R1, :grimacing:

Comment (carlosmmatos) [reply rhel7/profiles/stig_gui.profile:4]: hahaha my bad.. was looking at the wrong profile...whoops

**Review by ggbecker — APPROVED** [review]
LGTM. I hope profile id: `stig_gui` is good enough and will not require to be changed later :D

**Pull request merged**